core/api/views.py

```diff
@@ -0,0 +1,32 @@
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from core.api.serializers import GetForeignWarehouseSerializer
+from rest_framework import status
+
+from core.models import ForeignWarehouse
+
+
+@api_view(['GET'])
```

Comment:
Check class based views, but if you want, you still can use function based views